dipy/io/trackvis.py

```diff
@@ -5,11 +5,11 @@
 
 
 def load_trk(fname):
-    import nibabel.trackvis as tv
-    streams, hdr = tv.read(fname, points_space='rasmm')
-    return [i[0] for i in streams], hdr
-    # trkfile = nib.streamlines.load(fname)
-    # return trkfile.streamlines, trkfile.header
+    # import nibabel.trackvis as tv
+    # streams, hdr = tv.read(fname, points_space='rasmm')
+    # return [i[0] for i in streams], hdr
+    trkfile = nib.streamlines.load(fname)
```

Comment:
Need to use my latest Nibabel streamlines_compact branch to show 
https://dl.dropboxusercontent.com/u/24119444/standard.nii.gz
https://dl.dropboxusercontent.com/u/24119444/standard.trk